packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.tsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Prop, Element, Event, EventEmitter, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-usa-banner',
+  styleUrl: 'cbp-usa-banner.scss'
+})
+export class CbpUsaBanner {
+
+  @Element() host: HTMLElement;
+
+  /** When set, specifies that the banner is open */
+  @Prop({ reflect: true }) open: boolean;
+
+  // /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  // @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
```

Comment:
you can get rid of sx - it's not fully hooked up anyway in here.

Reply:
removed